src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,16 @@
 package mate.academy
 
+import mate.academy.service.FigureSupplier
+
 fun main() {
-    // draw all figures here
+    val figureSupplier = FigureSupplier()
+
+    listOf(
+            figureSupplier.getRandomFigure(),
```

Comment:
What if there would be 1M figures? :) 
Try to find more flexible solution without code duplication.

Reply:
done